packages/FileUpload/src/index.tsx

```diff
@@ -20,19 +22,22 @@ const FileUpload: React.SFC<IFileUploadProps> = ({
       files.map(
         file =>
           new Promise((resolve, reject) => {
+            const date = new Date();
             if (file.type && file.type.includes("image")) {
               const reader = new FileReader();
               reader.readAsDataURL(file);
               reader.onload = (e: any) =>
                 resolve({
                   base64: e.target.result,
+                  key: `${file.name}-${date.getTime()}`,
```

Comment:
Id param has already added in this component we are using uuid, the problem was that using time if you drop like 3 images at the same time and try to remove one of them the time is the same because has been added together. Bring all this changes from dev please.